mxcube3/routes/qutils.py

```diff
@@ -344,12 +349,17 @@ def set_dc_params(model, entry, task_data):
 
     acq.acquisition_parameters.set_from_dict(params)
     acq.path_template.set_from_dict(params)
+    acq.path_template.base_prefix = params['prefix']
 
     full_path = os.path.join(mxcube.session.get_base_image_directory(),
                              params.get('path', 'dummy_path'))
 
     acq.path_template.directory = full_path
 
+    process_path = os.path.join(mxcube.session.get_base_process_directory(),
+                                params.get('path', 'dummy_path'))
```

Comment:
Well each institute have their schema appending raw_data and processed_data might work in most cases. The risk is that its added twice.

But why append anything at all if path does not exist ?, and path should always exist. The user cant add a collection
without a valid path so this will later be validated before the user adds the collection.

I know that this dummy_path thing have been there for while, we can fix that at later state, we can let it pass 
for now. I should maybe have removed that myself when I did the refactoring but I forgot. But if you have the time
simply remove the second argument to get, or set it to ''.